specs/merge/validator.md

```diff
@@ -75,17 +75,12 @@ def get_pow_block_at_total_difficulty(total_difficulty: uint256, pow_chain: Sequ
     return None
 
 
-def compute_randao_mix(state: BeaconState, randao_reveal: BLSSignature) -> Bytes32:
-    epoch = get_current_epoch(state)
-    return xor(get_randao_mix(state, epoch), hash(randao_reveal))
-
-
 def produce_execution_payload(state: BeaconState,
                               parent_hash: Hash32,
                               randao_reveal: BLSSignature,
```

Comment:
Need to remove `randao_reveal` as an arg here and from the caller in `get_execution_payload`

Reply:
Good catch! Should be fixed now